### PR TITLE
ADEN-12355 short/long page

### PR DESCRIPTION
**Pull request by lnawrotek**

### CODE REVIEW

File: src/core/utils/targeting.ts [hunk 1]
```diff
@@ -80,6 +81,13 @@ class Targeting {
 			bundles.push('VIDEO_TIER_3_BUNDLE');
 		}
 
+		const wordCount = context.get('targeting.word_count') || -1;
+
+		if (wordCount <= shortPageWordsLimit) {
```
Comment (nandy-andy) [inline src/core/utils/targeting.ts:86]: I wonder if we shouldn't push `short_page` bundle only if we do have the word count and it's smaller or equals the limit 🤔 So, the condition would be `wordCount > -1 && wordCount <= shortPageWordsLimit`. This way we won't count some special case pages when the MediaWiki's Output won't return the HTML -- we'll focus mostly on the articles, I guess 🤔 But this is something worth putting in the QA notes - to check different page types.

Comment (lnawrotek) [reply src/core/utils/targeting.ts:86]: It's a good question. We have to decide which type is the default type. I think `wordCount > -1 && wordCount <= shortPageWordsLimit` is OK, we can focus on articles.

File: spec/core/utils/targeting.spec.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ describe('targeting', () => {
 					skin: ['fandom_desktop'],
 				},
 			}),
-		).to.deep.equal(['bundle2', 'bundle3']);
+		).to.deep.equal(['bundle2', 'bundle3', 'short_page']);
```
[the remark this answers is not in the document]
Comment (lnawrotek) [reply spec/core/utils/targeting.spec.ts:53]: I can remove the `short_page` bundle if I change condition https://github.com/Wikia/ad-engine/pull/1611#discussion_r1026263177

File: src/core/utils/targeting.ts (hunk 1 above)
Comment (nandy-andy) [reply src/core/utils/targeting.ts:86]: Let's do it this way then 👌

Comment (lnawrotek) [reply src/core/utils/targeting.ts]: I've added a unit test to check `word_count = -1`

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**